main/editor.js

```diff
@@ -81,8 +87,13 @@ const setOptions = options => {
 
 const getEditors = () => editors.values();
 
+ipc.answerRenderer('save-original', ({inputPath, outputPath}) => {
+  cpFile(inputPath, outputPath);
```

Comment:
You can just use the native method here. `fs.copyFile()`. Make sure you pass the `fs.constants.COPYFILE_FICLONE` to make copying fast.

Reply:
e3168b3